src/components/carousel/buttons/slideControl.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './slideControl.scss';
+import { BsArrowLeftSquare } from 'react-icons/bs';
+import { BsArrowRightSquare } from 'react-icons/bs';
+
+const SlideControl = ({navigateSlide, arrow}) => {
+    return (
+        <div className="carousel__navigation">
+            <button className="carousel__navigation__btn" onClick={navigateSlide}>
+                {arrow === "left" &&
+                    <BsArrowLeftSquare className="carousel__navigation__btn__arrow carousel__navigation__btn__arrow--left"/>
+                }
+                {arrow === "right" &&
+                    <BsArrowRightSquare className="carousel__navigation__btn__arrow carousel__navigation__btn__arrow--right"/>
+                }   
```

Comment:
It's can be easier.
`const SlideControl = ({navigateSlide, arrowModifier, Icon}) =>`

```
<Icon className={`carousel__navigation__btn__arrow carousel__navigation__btn__arrow--${arrowModifier}`}>
```